modules/ai/module.php

```diff
@@ -0,0 +1,314 @@
+<?php
+namespace Elementor\Modules\Ai;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Common\Modules\Connect\Module as ConnectModule;
+use Elementor\Modules\Ai\Connect\Ai;
+use Elementor\Plugin;
+use Elementor\User;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+class Module extends BaseModule {
+
+	public function get_name() {
+		return 'ai';
+	}
+
+	public function __construct() {
+		parent::__construct();
+
+		add_action( 'elementor/connect/apps/register', function ( ConnectModule $connect_module ) {
+			$connect_module->register_app( 'ai', Ai::get_class_name() );
+		} );
+
+		add_action( 'elementor/ajax/register_actions', function( $ajax ) {
+			$ajax->register_ajax_action( 'ai_get_user_information', [ $this, 'ajax_ai_get_user_information' ] );
+			$ajax->register_ajax_action( 'ai_get_completion_text', [ $this, 'ajax_ai_get_completion_text' ] );
+			$ajax->register_ajax_action( 'ai_get_edit_text', [ $this, 'ajax_ai_get_edit_text' ] );
+			$ajax->register_ajax_action( 'ai_get_custom_code', [ $this, 'ajax_ai_get_custom_code' ] );
+			$ajax->register_ajax_action( 'ai_get_custom_css', [ $this, 'ajax_ai_get_custom_css' ] );
+			$ajax->register_ajax_action( 'ai_set_get_started', [ $this, 'ajax_ai_set_get_started' ] );
+			$ajax->register_ajax_action( 'ai_set_status_feedback', [ $this, 'ajax_ai_set_status_feedback' ] );
+			$ajax->register_ajax_action( 'ai_create_images', [ $this, 'ajax_ai_create_images' ] );
+			$ajax->register_ajax_action( 'ai_upload_image', [ $this, 'ajax_ai_upload_image' ] );
+		} );
+
+		add_action( 'elementor/editor/before_enqueue_scripts', function() {
+			wp_enqueue_script(
+				'elementor-ai',
```

Comment:
```suggestion
				'elementor-ai-editor',
```

maybe? 
because you also have one for the admin